repo_health/index/static/client/src/components/issue-stats/issue-stats.component.js

```diff
@@ -0,0 +1,45 @@
+/*
+* issue-stats.component.js - (C) Copyright - 2017
+* This software is copyrighted to contributors listed in CONTRIBUTIONS.md.
+*
+* SPDX-License-Identifier: MIT
+*
+* Author(s) of this file:
+* J.Harding
+*
+* Create issue-stats component.
+*/
+
+import template from '../base-component/base.template';
+import BaseStatsComponent from '../base-component/base-stats.component';
+
+const issueStatsComponent = {
+  template,
+  bindings: {issueStatsUrl: '='},
+  controller: class issueStatsComponent extends BaseStatsComponent {
```

Comment:
I think you want to use a service versus extending another component. The Base component just does business logic stuff, which is what services are for.